<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRectangleForm</class>
 <widget class="QWidget" name="CreateRectangleForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>239</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>239</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>LeftDownPoint X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editLdpX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>LeftDownPoint Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editLdpY"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Lenth</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editLenth"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editWidth"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnEnterData">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Cnacle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
